packages/veritone-react-common/src/components/SimpleSearchBar/index.js

```diff
@@ -0,0 +1,227 @@
+                    // if we have plain string array
+                    // eslint-disable-next-line lodash/prefer-lodash-typecheck
+                    if (typeof result === 'string') {
+                      return (
+                        <MenuItem
+                          key={result}
+                          // eslint-disable-next-line react/jsx-no-bind
+                          onClick={() => clear() || result.onClick(result)}
+                        >
+                          {result}
+                        </MenuItem>
+                      );
+                    }
+                    // if we have data groups
+                    else {
+                      if (result.data && result.data.length) {
+                        // get the template
+                        let TemplateComponent = result.template;
+                        return (
+                          <Fragment key={Math.random()}>
+                            {// map data to template
+                            result.data.map(data => {
+                              // eslint-disable-next-line lodash/prefer-lodash-typecheck
+                              if (typeof data === 'string') {
+                                return (
+                                  <MenuItem
+                                    key={data}
+                                    // eslint-disable-next-line react/jsx-no-bind
+                                    onClick={() =>
+                                      clear() || result.onClick(data)
+                                    }
+                                  >
+                                    {TemplateComponent(data)}
+                                  </MenuItem>
+                                );
+                              } else {
+                                return (
+                                  <MenuItem
+                                    key={data.id}
+                                    // eslint-disable-next-line react/jsx-no-bind
+                                    onClick={() =>
+                                      clear() || result.onClick(data)
+                                    }
+                                  >
+                                    <TemplateComponent {...data} />
+                                  </MenuItem>
+                                );
+                              }
+                            })}
+                            <Divider />
+                          </Fragment>
+                        );
+                      }
+                    }
+                  })}
+                </div>
+              </Fragment>
+            )}
+          </PaperOrDiv>
+        </div>
+      </ClickAwayListener>
+    );
+  }
+}
+
+SimpleSearchBarBase.propTypes = {
+  autocomplete: bool,
+  onChange: func,
+  onSubmit: func,
+  value: string,
+  isLoading: bool,
+  placeholder: string,
+  autocompleteResults: arrayOf(any),
```

Comment:
Should spec out the expected shape?

Reply:
Correct, resolved here; https://github.com/veritone/veritone-sdk/pull/382/commits/2c4fa7c07fff7dee90a89b5cbc285494f61dd409